shared/desktop/remote/store.desktop.js

```diff
@@ -41,10 +42,32 @@ class RemoteStore {
     sendToMainWindow('remoteWindowWantsProps', props.windowComponent, props.windowParam)
   }
 
+  // Some shared inner components needs immutable structures (Avatar), we can likely fix this longer term but for now lets just
```

Comment:
let me know if this comment makes sense. quick overview:

we want our remote windows to be simple.
our 'old' style was to make a subset of the store which we transport over the wire. this was pretty inefficient as we'd often in mapStateToProps reduce this down to a much smaller set of props.
instead of that we essentially run mapStateToProps on the mainWindow and transport the plain props over the wire after this processing.

This works great assuming you don't have further inner connected components.

We already had a special connector for handling avatar requests, and now avatar handling is slightly extended to support showing the followers/following